src/store/channels/index.ts

```diff
@@ -53,7 +53,7 @@ export interface Channel {
   hasLoadedMessages: boolean;
   conversationStatus: ConversationStatus;
   messagesFetchStatus: MessagesFetchState;
-  admin: string;
+  admins: string[];
```

Comment:
At a glance you'd never know if this was actual objects of some type or not. `adminIds` or, probably better, `adminMatrixIds` since it's not obvious whether it's a user id or a matrix id.

Reply:
ok